scripts/updateQDKVersion.ps1

```diff
@@ -36,7 +39,7 @@ $csFiles | ForEach-Object {
     } | Set-Content -Encoding UTF8 $_.Path
 }
 
-$ipynbString = '%package Microsoft.Quantum.Katas::(?<oldVersion>[0-9|\.]+)'
+$ipynbString = '%package Microsoft.Quantum.Katas::' + $versionRegex
```

Comment:
Minor, but you should be able to use string interpolation here to avoid string concatenation.
```suggestion
$ipynbString = "%package Microsoft.Quantum.Katas::$versionRegex"
```

Reply:
I tried to use string interpolation with csproj line, and it didn't work, so I decided I got the wrong syntax and dropped the idea...  Do you know why this works in three strings but not the fourth one?

Reply:
I think this is a difference between `'` and `"`:
```PowerShell
PS tmp> $x = "foo"
PS tmp> "${x}bar"
foobar
PS tmp> '${x}bar'
${x}bar
```

Reply:
No, with `'` it runs but doesn't give the result I want, but with `"` it threw an exception. Something more interesting I think...